spec/javascripts/packs/backup-code-analytics-spec.js

```diff
@@ -0,0 +1,30 @@
+import { screen } from '@testing-library/dom';
+import userEvent from '@testing-library/user-event';
+import { useSandbox } from '@18f/identity-test-helpers';
+
+describe('backupCodeAnalytics', () => {
+  const sandbox = useSandbox();
+
+  beforeEach(() => {
+    document.body.innerHTML = `
+      <a download="backup_codes.txt" class="usa-button usa-button--outline" href="#"><svg aria-hidden="true" focusable="false" role="img" class="usa-icon">
+  <use href="http://test.host/assets/identity-style-guide/dist/assets/img/sprite-8eff3bf787e3ce0eab960fe5e9eccf4418d9af6a9f8c95a9ec9254aa778b2dbd.svg#file_download"></use>
+</svg>Download</a>
+    `;
+    import('../../../app/javascript/packs/backup-code-analytics.ts');
```

Comment:
This could cause a race condition, since the import is asynchronous, and the tests may begin to run before the import is complete.

The `import` returns a promise, which we can either `await` or return (Mocha will wait if the return value is a promise).

```suggestion
    await import('../../../app/javascript/packs/backup-code-analytics.ts');
```

```suggestion
    return import('../../../app/javascript/packs/backup-code-analytics.ts');
```